src/cloud-code/httpRequest.js

```diff
@@ -82,10 +82,63 @@ const encodeBody = function ({ body, headers = {} }) {
  *
  * @method httpRequest
  * @name Parse.Cloud.httpRequest
- * @param {Parse.Cloud.HTTPOptions} options The Parse.Cloud.HTTPOptions object that makes the request.
- * @return {Promise<Parse.Cloud.HTTPResponse>} A promise that will be resolved with a {@link Parse.Cloud.HTTPResponse} object when the request completes.
+ * @param {Object} options axios object for options
+ * @return {Promise<Object>} axios response object
  */
-module.exports = function httpRequest(options) {
+import axios from 'axios';
```

Comment:
Shouldn't `Parse.Cloud.httpRequest` be removed entirely? We aren't offering this method anymore to developers. Or is used by Parse Server internally?